app/helpers/conditionalHTML.js

```diff
@@ -10,7 +10,7 @@ const getHtml = (label, labelData, fieldValueData, error) => {
             ${labelData}
           </label>
           <div class="govuk-input__wrapper">
-            <input class="govuk-input govuk-input--width-4 govuk-input--error" autocomplete="off" id="${label}" name="${label}" value="${fieldValue}">
+            <input class="govuk-input govuk-input--width-4" autocomplete="off" id="${label}" name="${label}" value="">
```

Comment:
how ill it hold the value if the user come back in the journey ?? 
also is there any validation on this ??

Reply:
validation is all done in the question bank. We've made sure to throughly test it and it all works

Reply:
As for saving value, yes thats the only issue

if value is there, it is saved but it causes many many _many_ sidebar issues (it adds to the sidebar every time the user enters a value, no matter whether it is correct or not)

i can try and work on it now, but i've so far been unsuccessful with it

Reply:
Fixed the weird bug the value causes. This can now be resolved